frappe/commands/site.py

```diff
@@ -72,11 +88,22 @@ def new_site(
 	"Create a new site"
 	from frappe.installer import _new_site
 
+	frappe.init(site=site, new_site=True)
+
+	if no_mariadb_socket:
+		click.secho(
+			"--no-mariadb-socket is DEPRECATED; "
+			"use --mariadb-user-host-login-scope='%' (wildcard) or --mariadb-user-host-login-scope=<myhostscope>, instead. "
+			"The name of this option was misleading: it had nothing to do with sockets.",
+			fg="yellow",
+		)
+		frappe.flags.mariadb_user_host_login_scope = "%"
```

Comment:
Missed something earlier, so when creating the config frappe.init() is called again, so this flag doesn't persist.
Will let the passing as parameter continue, the rest seems fine.